examples/mechanics/kalthoff_winkler.cpp

```diff
@@ -76,23 +76,26 @@ void kalthoffWinklerExample( const std::string filename )
     CabanaPD::Prenotch<2> prenotch( v1, v2, notch_positions );
 
     // ====================================================
-    //                    Force model
+    //                    Force model options
     // ====================================================
-    using model_type = CabanaPD::ForceModel<CabanaPD::PMB, CabanaPD::Fracture>;
-    model_type force_model( delta, K, G0 );
-    // using model_type =
-    //     CabanaPD::ForceModel<CabanaPD::LPS, CabanaPD::Fracture>;
-    // model_type force_model( delta, K, G, G0 );
+    using model_type = CabanaPD::PMB;
+    using thermal_type = CabanaPD::TemperatureIndependent;
 
     // ====================================================
     //                 Particle generation
     // ====================================================
     // Does not set displacements, velocities, etc.
     auto particles = std::make_shared<
-        CabanaPD::Particles<memory_space, typename model_type::base_model,
-                            typename model_type::thermal_type>>(
+        CabanaPD::Particles<memory_space, model_type, thermal_type>>(
         exec_space(), low_corner, high_corner, num_cells, halo_width );
 
+    std::array<double, 2> same_d = { delta, delta };
```

Comment:
We shouldn't modify the Kalthoff Winkler example. If this is just for testing, we need to create a new example.

Reply:
I agree, I will remove when you push a new example